pkg/matcher/http/absmatch.go

```diff
@@ -163,7 +164,7 @@ func CompareHTTPReq(tcs1, tcs2 *models.TestCase, _ models.GlobalNoise, ignoreOrd
 	tcs1.HTTPReq.Header["Keploy-Test-Set-Id"] = "dummyTestSet"
 
 	// compare http req headers
-	ok := CompareHeaders(pkg.ToHTTPHeader(tcs1.HTTPReq.Header), pkg.ToHTTPHeader(tcs2.HTTPReq.Header), &reqCompare.HeaderResult, reqHeaderNoise)
+	ok := matcherUtils.CompareHeaders(pkg.ToHTTPHeader(tcs1.HTTPReq.Header), pkg.ToHTTPHeader(tcs2.HTTPReq.Header), &reqCompare.HeaderResult, reqHeaderNoise)
```

Comment:
You could have just called this matcher don't use matcherUtils when it is not an actual util that you referred.